src/constants/description/description.ts

```diff
@@ -32,4 +32,15 @@ export const description = {
       account: 'konto',
       forgotPass: 'zapomniałeś hasła ?',
    },
+   studentPortfolio: {
+      rating: 'Oceny',
+      expectation: 'Oczekiwanie w stosunku do zatrudnienia',
+      education: 'Edukacja',
+      courses: 'Kursy',
+      experience: 'Doświadczenie zawodowe',
+      portfolio: 'Portfolio',
+      groupProject: 'Projekt w zespole scrumowym',
+      ownProject: 'Projekt na zaliczenie',
+      aboutMe: 'O mnie',
+   },
```

Comment:
In render component try use ::firstLetter -> upperCase.  Someone can write a lowercase letter and it will be lowercase wherever it is used. ;)